music21/scale/__init__.py

```diff
@@ -2830,9 +2892,9 @@ class HarmonicMinorScale(DiatonicScale):
     '''
     The harmonic minor collection, realized as a scale.
 
-    (The usage of this collection as a scale, is quite ahistorical for
+    (The usage of this collection as a scale is quite ahistorical for
     Western European classical music, but it is common in other parts of the
-    world, but where the term "HarmonicMinor" would not be appropriate).
```

Comment:
thanks for the space, but the "but" should remain

Reply:
Right, because "where" is not delimiting certain parts of the world where the term is/is not meaningful.